UnityProject/Assets/Scripts/Items/Implants/Organs/Stomach.cs

```diff
@@ -36,6 +37,15 @@ public override void ImplantPeriodicUpdate()
 				RelatedPart.BloodContainer.Add(Digesting);
 			}
 
+			if (StomachContents.CurrentReagentMix.MajorMixReagent == Salt && StomachContents.CurrentReagentMix.Total > 15)
```

Comment:
wouldn't it be better to have this in the bloodstream? vs stomach

Reply:
Done.